nextcord/message.py

```diff
@@ -543,6 +544,62 @@ def flatten_handlers(cls):
     return cls
 
 
+class _MessageInteractionOptional(TypedDict, total=False):
+    member: Member
+
+
+class MessageInteractionPayload(_MessageInteractionOptional):
+    id: Snowflake
+    type: InteractionType
+    name: str
+    user: User
```

Comment:
Don't put any typed dicts here. They should be in `nextcord.types` where they were before.